Automattic/jetpack
# Lazy Images: remove module

## Proposed changes:

The module was deprecated in #33782. We can now safely remove the module from the plugin.

### Other information:


## Jetpack product discussion

* #33049

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* No functionality should change, since the module was already removed a few releases ago. 
* Go to Jetpack > Settings > Modules
    * You should not see any "Lazy Images" module in the list.
